Javascript/100-Problems/script.js

```diff
@@ -463,3 +464,185 @@ const avgMarkForSubjects = () => {
 };
 
 console.log(avgMarkForSubjects());
+
+//21: Write a function to find and print the subject with the highest total marks.
+
+const highestTotalMark = () => {
+  let highestTotal = 0;
+  let markArray = totalMarksForSubjects();
+
+  markArray.map((mark) => {
```

Comment:
we can do it like this also

`const highestTotalMark = () => {
  let highestTotal = 0;
  let subjects = []
  let markArray = totalMarksForSubjects();
  
  markArray.map((mark) => {
    if (highestTotal < mark.total) {
      subjects = [{ sub: mark.subject, mark: mark.total }]
      highestTotal = mark.total
    } else if (highestTotal === mark.total) subjects.push({ sub: mark.subject, mark: mark.total })
  });

  return subjects;
};`

just for reference